src/plugins/intel_gpu/src/plugin/ops/constant.cpp

```diff
@@ -185,6 +185,10 @@ void createClDnnConstant(Program& p, const ngraph::Shape& constDims, const std::
         constTensor = getConstTensor(newDims);
     }
 
+    if (p.use_new_shape_infer() && newDims.empty()) {
+        newDims.push_back(1);
```

Comment:
I have a feeling that this may lead to shape infer issues as scalar rank is 0 and this may be important for some ops.

Reply:
Fixed